islpy/__init__.py

```diff
@@ -935,13 +935,20 @@ def my_ismethod(class_, method_name):
     for args_triple in [
             (BasicSet, Set, Set.from_basic_set),
             (BasicMap, Map, Map.from_basic_map),
+            (BasicSet, UnionSet, lambda x: UnionSet.from_set(Set.from_basic_set(x))),
```

Comment:
Could you also add these to the docs} https://documen.tician.de/islpy/reference.html#automatic-casts